src/test/java/com/gizasystems/automationexercise/tests/SearchProducts/SearchedProductsApisTest.java

```diff
@@ -0,0 +1,33 @@
+package com.gizasystems.automationexercise.tests.SearchProducts;
+import com.gizasystems.automationexercise.apis.Apis;
+import com.gizasystems.automationexercise.apis.ApisSearchProducts;
+import com.shaft.driver.SHAFT;
+import jdk.jfr.Description;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class SearchedProductsApisTest {
+
+    private SHAFT.API api;
+    private SHAFT.TestData.JSON testData;
+
+
+    @BeforeClass
+    public void beforeClass() {
+        testData = new SHAFT.TestData.JSON("src/test/resources/testDataFiles/SearchProductApi.json");
+    }
+
+    @BeforeMethod
+    public void beforeMethod() {
+        api = new SHAFT.API(Apis.ApisBaseUrl);
+    }
```

Comment:
Configuration methods should be at the bottom of the test class (after the tests)